kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -63,10 +65,12 @@
               <KIcon icon="disconnected" />
             </span>
             &nbsp;&nbsp;
-            <span data-test="no-other-label">{{ injectedtr('noOtherLibraries') }}</span>
+            <span  data-test="no-other-label">{{ injectedtr('noOtherLibraries') }}</span>
           </span>
         </div>
       </KGridItem>
+    </KGridItem>
```

Comment:
It seems that this `KGridItem` is placed within another one, but the otherLibraries text should be within its own `KGridItem`, separate from the grid where the connection status is located, like this:
```
<KGridItem>
  <h1> otherLibraries text </h1>
</KGridItem>
<KGridItem>
  <div class="sync-status">
  </div>
</KGridItem>
```